porcupine/plugins/run/terminal.py

```diff
@@ -24,9 +24,13 @@
 
 
 # getting this to work in powershell turned out to be hard :(
-def _run_in_windows_cmd(command: str, cwd: Path, env: dict[str, str]) -> None:
+def _run_in_windows_cmd(command: str | None, cwd: Path, env: dict[str, str]) -> None:
     log.debug("using Windows command prompt")
 
+    if not command:
+        subprocess.Popen(["cmd", "/K", f"cd /d {shlex.quote(str(cwd))}"], env=env)
```

Comment:
`shlex.quote` uses single quotes which doesn't work on windows. But `subprocess.list2cmdline` is a Windows-specific (undocumented) function that does this:

```
>>> shlex.quote('foo bar')
"'foo bar'"
>>> subprocess.list2cmdline(['cd', '/d', 'foo bar'])
'cd /d "foo bar"'
```

```suggestion
        cd_command = subprocess.list2cmdline(["cd", "/d", str(cwd)])
        subprocess.Popen(["cmd", "/K", cd_command], env=env)
```